<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>579</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>869</width>
    <height>579</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Quizz</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>901</width>
      <height>581</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>901</width>
      <height>0</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>901</width>
      <height>591</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #f5f5f5;</string>
    </property>
    <widget class="QWidget" name="pageLogin">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>411</width>
        <height>291</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="title">
       <string>QIUZZ SYSTEM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="layoutUsername">
         <item>
          <widget class="QLabel" name="lbUsername">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtUsername">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutPassword">
         <item>
          <widget class="QLabel" name="lbPassword">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtPassword">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btnLogin">
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: #4CAF50;
        color: white;
        font: bold 14px 'Arial';
        border: none;
        border-radius: 5px;
        padding: 10px;
        margin-top: 10px;
    }

    /* Khi di chuột qua nút đăng nhập */
    QPushButton:hover {
        background-color: #45a049;
    }

    /* Khi nhấn vào nút đăng nhập */
    QPushButton:pressed {
        background-color: #388E3C;
    }</string>
         </property>
         <property name="text">
          <string>Login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRegister">
         <property name="styleSheet">
          <string notr="true">QPushButton {
        background-color: #0059ff;
        color: white;
        font: bold 14px 'Arial';
        border: none;
        border-radius: 5px;
        padding: 10px;
        margin-top: 10px;
    }

    /* Khi di chuột qua nút đăng nhập */
    QPushButton:hover {
        background-color: #0037ff;
    }

    /* Khi nhấn vào nút đăng nhập */
    QPushButton:pressed {
        background-color: #2600ff;
    }</string>
         </property>
         <property name="text">
          <string>Register</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="pageTeacherDashboard">
     <widget class="QGroupBox" name="leftSidebar">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>-30</y>
        <width>221</width>
        <height>611</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #2D2D3B;</string>
      </property>
      <property name="title">
       <string>GroupBox</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>221</width>
         <height>183</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="btnGroup">
        <item>
         <widget class="QPushButton" name="btnDashboard">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2D2D3B;
    color: #FFFFFF;
    border: none;
    padding: 10px;
    text-align: center;
}
QPushButton:hover {
    background-color: #444457; /* Hiệu ứng hover */
    border-left: 3px solid #FF4757; /* Đường highlight màu đỏ */
}</string>
          </property>
          <property name="text">
           <string>Dashboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnManageQuizz">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2D2D3B;
    color: #FFFFFF;
    border: none;
    padding: 10px;
    text-align: center;
}
QPushButton:hover {
    background-color: #444457; /* Hiệu ứng hover */
    border-left: 3px solid #FF4757; /* Đường highlight màu đỏ */
}</string>
          </property>
          <property name="text">
           <string>Manage Quizz</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnEditProfile">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2D2D3B;
    color: #FFFFFF;
    border: none;
    padding: 10px;
    text-align: center;
}
QPushButton:hover {
    background-color: #444457; /* Hiệu ứng hover */
    border-left: 3px solid #FF4757; /* Đường highlight màu đỏ */
}</string>
          </property>
          <property name="text">
           <string>Edit Profile</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnLogoutTeacherDashboard">
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #2D2D3B;
    color: #FFFFFF;
    border: none;
    padding: 10px;
    text-align: center;
}
QPushButton:hover {
    background-color: #444457; /* Hiệu ứng hover */
    border-left: 3px solid #FF4757; /* Đường highlight màu đỏ */
}</string>
          </property>
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="lbTeacherName">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>300</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #FFFFFF;
    font-size: 20px;
    margin: 10px;
text-align: center;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Teacher 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>0</y>
        <width>651</width>
        <height>581</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>741</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: #B63DF2;
    border: 2px solid #B63DF2;
color: white;
    font-size: 18px;
    font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Dashboard</string>
       </property>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>60</y>
         <width>471</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="stat">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="styleSheet">
           <string notr="true">color: #2D2D3B;
font-size: 28px;
font-weight: bold;
text-align: center;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:17pt;&quot;&gt;3&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="styleSheet">
           <string notr="true">color: #2D2D3B;
    font-size: 28px;
    font-weight: bold;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:17pt;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="styleSheet">
           <string notr="true">color: #2D2D3B;
    font-size: 28px;
    font-weight: bold;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:17pt;&quot;&gt;1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>160</y>
         <width>471</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="statInfo">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="styleSheet">
           <string notr="true">color: #555555;
    font-size: 14px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Completed Quizzes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="styleSheet">
           <string notr="true">color: #555555;
    font-size: 14px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Running Quizzes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="styleSheet">
           <string notr="true">color: #555555;
    font-size: 14px;</string>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Upcoming Quizzes&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableWidget" name="tbTeacherDashboard">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>240</y>
         <width>601</width>
         <height>311</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QTableWidget {
    background-color: #FFFFFF; /* Màu nền của bảng */
    border: 2px solid #B63DF2; /* Viền của bảng */
    gridline-color: #CCCCCC; /* Màu đường kẻ */
    font-size: 14px; /* Kích thước chữ */
    color: #333333; /* Màu chữ */
    selection-background-color: #B63DF2; /* Màu nền khi được chọn */
    selection-color: #FFFFFF; /* Màu chữ khi được chọn */
}

QHeaderView::section {
    background-color: #B63DF2; /* Màu nền tiêu đề */
    color: #FFFFFF; /* Màu chữ của tiêu đề */
    padding: 8px; /* Khoảng cách bên trong tiêu đề */
    font-weight: bold; /* In đậm chữ */
    border: 1px solid #B63DF2; /* Viền tiêu đề */
}

QTableWidget::item {
    border-bottom: 1px solid #CCCCCC; /* Đường kẻ dưới mỗi ô */
    padding: 5px; /* Khoảng cách bên trong ô */
}

QTableWidget::item:selected {
    background-color: #B63DF2; /* Màu nền khi ô được chọn */
    color: #FFFFFF; /* Màu chữ khi ô được chọn */
}

QTableCornerButton::section {
    background-color: #B63DF2; /* Màu nền góc trên cùng bên trái */
    border: 1px solid #B63DF2; /* Viền của góc */
}</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="pageRegister">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>130</y>
        <width>411</width>
        <height>311</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: black;</string>
      </property>
      <property name="title">
       <string>QIUZZ SYSTEM</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignTop</set>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="layoutFullname">
         <item>
          <widget class="QLabel" name="lbFullname">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Full name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtFullname">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutUsername_2">
         <item>
          <widget class="QLabel" name="lbUsername_2">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Username:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtUsername_2">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutPassword_2">
         <item>
          <widget class="QLabel" name="lbPassword_2">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Password:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txtPassword_2">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutChoice">
         <item>
          <widget class="QLabel" name="lbChoice">
           <property name="styleSheet">
            <string notr="true">color: #333;
font: 14px 'Arial';
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbChoice">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
        background-color: #fff;
        border: 1px solid #ccc;
        border-radius: 5px;
        padding: 8px;
        font-size: 14px;
        color: #333;
    }
    
    /* Khi ô nhập liệu được chọn */
    QLineEdit:focus {
        border: 1px solid #4CAF50;
    }</string>
           </property>
           <item>
            <property name="text">
             <string>Student</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Teacher</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="layoutButton">
         <item>
          <widget class="QPushButton" name="btnBack">
           <property name="styleSheet">
            <string notr="true">QPushButton {
        background-color: #fbfbff;
        color: #27ff64;
        font: bold 14px 'Arial';
        border: none;
        border-radius: 5px;
        padding: 10px;
        margin-top: 10px;
    }

    /* Khi nhấn vào nút đăng nhập */
    QPushButton:pressed {
        background-color: #27ff64;
		color: #fbfbff;
    }</string>
           </property>
           <property name="text">
            <string>Back to Login</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnRegister_2">
           <property name="styleSheet">
            <string notr="true">QPushButton {
        background-color: #0059ff;
        color: white;
        font: bold 14px 'Arial';
        border: none;
        border-radius: 5px;
        padding: 10px;
        margin-top: 10px;
    }

    /* Khi di chuột qua nút đăng nhập */
    QPushButton:hover {
        background-color: #0037ff;
    }

    /* Khi nhấn vào nút đăng nhập */
    QPushButton:pressed {
        background-color: #2600ff;
    }</string>
           </property>
           <property name="text">
            <string>Register</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
